miio/airhumidifier_jsq.py

```diff
@@ -43,19 +90,21 @@ class LedBrightness(enum.Enum):
     High = 2
 
 
-class AirHumidifierStatus(DeviceStatus):
-    """Container for status reports from the air humidifier jsq."""
+class OperationModeJsq002(enum.Enum):
+    Level1 = 1
+    Level2 = 2
+    Level3 = 3
 
-    def __init__(self, data: Dict[str, Any]) -> None:
-        """Status of an Air Humidifier (shuii.humidifier.jsq001):
 
-            [24, 30, 1, 1, 0, 2, 0, 0, 0]
+class LedBrightnessJsq002(enum.Enum):
+    Off = 1
+    Low = 2
+    High = 3
 
-        Parsed by AirHumidifierJsq device as:
-            {'temperature': 24, 'humidity': 29, 'mode': 1, 'buzzer': 1,
-             'child_lock': 0, 'led_brightness': 2, 'power': 0, 'no_water': 0,
-              'lid_opened': 0}
-        """
+
+class AirHumidifierStatusJsqCommon(DeviceStatus):
```

Comment:
Please extend the existing status class, the values that are not available for a model should be returned as `None`.